controller/array_action/array_mediator_ds8k.py

```diff
@@ -29,9 +30,9 @@
 ERROR_CODE_ALREADY_FLASHCOPY = '000000AE'
 ERROR_CODE_VOLUME_NOT_FOUND_OR_ALREADY_PART_OF_CS_RELATIONSHIP = '00000013'
 
-FLASHCOPY_PERSISTENT_OPTION = "persistent"
-FLASHCOPY_NO_BACKGROUND_COPY_OPTION = "no_background_copy"
-FLASHCOPY_PERMIT_SPACE_EFFICIENT_TARGET = "permit_space_efficient_target"
+FLASHCOPY_PERSISTENT_OPTION = ds8k_types.DS8K_OPTION_PER
+FLASHCOPY_NO_BACKGROUND_COPY_OPTION = ds8k_types.DS8K_OPTION_NBC
+FLASHCOPY_PERMIT_SPACE_EFFICIENT_TARGET = ds8k_types.DS8K_OPTION_PSET
```

Comment:
I know it's long, but consider adding `_OPTION`, for consistency

Reply:
done